Makefile

```diff
@@ -240,33 +256,45 @@ docker-login:
 docker-tag-dev:
 	@docker tag $(IMG) $(REPOSITORY):$(DEV_TAG)
 	@docker tag $(IMG) $(REPOSITORY):dev
+	@docker tag $(CRD_IMG) $(CRD_REPOSITORY):$(DEV_TAG)
+	@docker tag $(CRD_IMG) $(CRD_REPOSITORY):dev
 
 # Tag for Dev
 docker-tag-release:
 	@docker tag $(IMG) $(REPOSITORY):$(VERSION)
+	@docker tag $(CRD_IMG) $(CRD_REPOSITORY):$(VERSION)
 
 # Push for Dev
 docker-push-dev: docker-tag-dev
 	@docker push $(REPOSITORY):$(DEV_TAG)
 	@docker push $(REPOSITORY):dev
+	@docker push $(CRD_REPOSITORY):$(DEV_TAG)
```

Comment:
do we want to cut a new crd image for every commit to master?

Reply:
We need it because in the chart, the gatekeeper-crd image uses the same release tag as the gatekeeper image: 
```
containers:
      - name: crds-upgrade
        image: '{{ .Values.image.crdRepository }}:{{ .Values.image.release }}'
```